doc/manuals/user/cors.md

```diff
@@ -68,12 +71,12 @@ This header should be present in Orion's response for every `OPTIONS` request
 made to `/v2` resources. The user is free to set a value for the maximum time

```

Comment:
> The user is free to set a value for the maximum time (in seconds) a preflight request made to Orion can be cached by the client.

This is a bit confusing. Since you have already created a PR, may I ask you to change this line as well? Please see my suggestion below:

The user is free to set a value for the maximum time (in seconds) a client is allowed to cache a preflight request made to Orion.

Reply:
Fixed in 76ab66be28d95b9192b08e77a1120ffdacf1fee3